cachito/web/api_v1.py

```diff
@@ -29,8 +29,19 @@
 from cachito.web.utils import deep_sort_icm, pagination_metadata, str_to_bool
 from cachito.workers import tasks
 
+from prometheus_client import multiprocess
+from prometheus_client.core import CollectorRegistry
+from prometheus_flask_exporter import PrometheusMetrics
+
+
 api_v1 = flask.Blueprint("api_v1", __name__)
 
+# Prometheus Flask Exporter.
+#   - gets us our metrics.
+registry = CollectorRegistry()
+multiprocess.MultiProcessCollector(registry, path="/tmp/wsgi")
```

Comment:
I see envvar `PROMETHEUS_MULTIPROC_DIR`  everywhere. Why is it here hardcoded?

Reply:
Cleaned up.